.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
If you want eslint, you probably want the tooling one or apply maintainers to everything ;)